src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -867,12 +865,8 @@ def test_bc_kernel_field_only(monkeypatch, annexed, dist_mem):
     # function which we create using lambda.
     monkeypatch.setattr(arg, "ref_name",
                         lambda function_space=None: "vspace")
-    with pytest.raises(VisitorError) as excinfo:
+    with pytest.raises(VisitorError):
         _ = psy.gen
```

Comment:
Could we still check that it's the right VisitorError or does it not have a message associated with it?

Reply:
Ups, this shouldn't have been deleted, now is back